Update treeview_iteminvoked.md

dan-meier opened the pull request:
More explicitly defines when this event is invoked.

dan-meier committed:
Update treeview_iteminvoked.md

More explicitly defines when this event is invoked.

PRMerger15 commented:
@dan-meier : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft committed:
Update treeview_iteminvoked.md

Revised somewhat: Due to the variety of input devices that WIndows supports, we typically don't enumerate the interaction types (keyboard input being one example that isn't covered by your list). I reverted that change and moved the qualification to the remarks.

PRMerger19 commented:
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft commented:
#sign-off
